source/agora/node/FullNode.d

```diff
@@ -867,4 +868,26 @@ public class FullNode : API
         }
         return found_txs;
     }
+
+    /***************************************************************************
+
+        Params:
+            height = Height of block
+
+        Returns:
+            BlockHeaderSignature if the client has that block
+
+    ***************************************************************************/
+
+    public BlockHeaderSignature[] getBlockSignature (ulong height) @safe
+    {
+        BlockHeaderSignature[] block_signatures;
+        if (height <= this.getBlockHeight())
+        {
+            auto block = this.ledger.getBlockAtHeight(Height(height));
+            block_signatures = [ BlockHeaderSignature(Height(height), block.header.signature,
+                block.header.validators) ];
+        }
+        return block_signatures;
+    }
```

Comment:
Why does it return an array if we only ever return one value ?

Reply:
Now it return multiple values.